spec/relativetimeformat.html

```diff
@@ -30,9 +30,9 @@ <h1>InitializeRelativeTimeFormat ( _relativeTimeFormat_, _locales_, _options_ )<
         1. Set _opt_.[[nu]] to _numberingSystem_.
         1. Let _localeData_ be %RelativeTimeFormat%.[[LocaleData]].
         1. Let _r_ be ResolveLocale(%RelativeTimeFormat%.[[AvailableLocales]], _requestedLocales_, _opt_, %RelativeTimeFormat%.[[RelevantExtensionKeys]], _localeData_).
-        1. Let _locale_ be _r_.[[Locale]].
+        1. Let _locale_ be _r_.[[locale]].
```

Comment:
I don't love the fact that we have meaningfully different internal slots `[[Locale]]` and `[[locale]]`. This is correct but could use some improvement.